website/docs/r/dns_record_set.markdown

```diff
@@ -34,15 +44,25 @@ resource "google_dns_managed_zone" "prod" {
   name     = "prod-zone"
   dns_name = "prod.mydomain.com."
 }
+```
 
-resource "google_dns_record_set" "frontend" {
+### Adding a SPF record
+
+`\"` must be added around your `rrdatas` for a SPF record. Otherwise `rrdatas` string gets splitted on spaces.
```

Comment:
I think what we have as fine, and if we keep seeing the problem, we can escalate to the higher-visibility **Note:** block :)